ginga/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = 'ginga' %}
 {% set version = '2.5.20160627100500' %}
```

Comment:
Yes absolutely. The python-reported version will be different but its a small price to pay if we're basing releases on git hashes.